detox/src/utils/fsext.js

```diff
@@ -19,7 +19,12 @@ async function getDirectories (rootPath) {
   return dirs.sort();
 }
 
+function readdirSync(dirPath) {
+  return fs.readdirSync(dirPath);
+}
```

Comment:
I might be abusing code-styling here a bit, but what about:
```diff
-function readdirSync(dirPath) {
-  return fs.readdirSync(dirPath);
-}
+const readdirSync = fs.readdirSync;
```
?

Reply:
Mmm, the point was to ensure it can be safely mocked. But I guess you're right.

Reply:
Fixed in #3108